omp/__init__.py

```diff
@@ -25,6 +26,26 @@ class OpenMP(object):
     """
 
     def __init__(self):
+        if c_compiler is None or c_compiler.compiler_type != 'msvc':
```

Comment:
I don't like that. If ``c_compiler.compiler_type == 'msvc'``, what is the type : value of ``c_compiler`` ?